index.js

```diff
@@ -68,7 +75,17 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+
+const findById = (movies, id) => {
+  if (!movies.length) throw 'ERROR: Movies array is empty!';
+  let movieID = null;
+  for (let movie of movies){
+    if (movie.imdbID === id){
+      return movies.find((movie) => movie.imdbID === id);
+    }
+  }
+  return movieID;
```

Comment:
So instead here we could have done:

`let movieID = movies.find((movie) => movie.imdbID === id);
return movieID === undefined ? null: movieID`